x/cosmwasmpool/README.md

```diff
@@ -0,0 +1,39 @@
+# CosmWasm Pool Module
```

Comment:
Note: @iboss-ptk I'm down to eventually overwrite/merge this with yours https://github.com/osmosis-labs/osmosis/pull/4810, and can help with resolving conflicts